src/Controllers/OaiController.php

```diff
@@ -105,7 +111,6 @@ protected function BadVerbResponse(HTTPRequest $request): HTTPResponse
         $requestUrl = sprintf('%s%s', Director::absoluteBaseURL(), $request->getURL());
```

Comment:
```suggestion
```
It is not related to this PR, but I found there is a [getRequestUrl](https://github.com/silverstripe-terraformers/open-archive-initiative-repository/blob/229c084a83eee6ca241e5947a8d31544eef15046/src/Controllers/OaiController.php#L310) method we can use to replace this code where the `$requestUrl` is 'manually' generated.

Reply:
Oh lol, I forgot about that. Thanks!